.spelling

```diff
@@ -142,94 +142,24 @@ accessMembers.png
 protectedMainBranch.png
 fullstory
 uncheck
-tripple-dot
-jira
-popup
-Solvi
-Stefania
-deeplinks
-checkbox
-bool
-setAvoLogger
-setSystemProperties
-FacebookAnalytics
-Permutive
-ZendeskConnect
-Apptelemetry
-Rudderstack
-Freshpaint
-Posthog
-Kissmetrics
-LaunchDarkly
-Pendo
-Fivetran
-setUserId
-logRevenue
-logRevenueV2
-enums
-7d
-timewindow
-csv
-Avo.initAvo
-property_presence_per_event.png
-property_presence_per_source_and_event.png
-publishing.svg
-200px
-auto_publishing.png
-publish_filter.png
-publishing_protocols.svg
-publishing_lexicon.svg
-publishing_govern.svg
-i.e.
-erroring
-publishing_webhook.svg
-metricId
-sourceId
-destinationId
-categoryId
-publishInfo
-PublishInfo
-PropertyRules
-PropertyRule
-EventSegmentation
-eventId
-whereFilter
-groupBy
-groupByFilter
-propertyId
-programmingLanguage
-developmentplatform
-branchid
-branchname
-integrationid
-integrationname
-publishdate
-publishmethod
-branchmerge
-jitpack.io
-logcat
-nullable
-debuggermanager
-avoinspector
-cocoapods
-nsdictionary
-analyticsdebugger
-propname
-camelCase
```

Comment:
Hm, that's why it's failing I guess, some words were removed from our custom dictionary

Reply:
Ahhh I see! I probably messed up the merge with the main branch 😬